source/FreeRTOS_TCP_IP.c

```diff
@@ -608,7 +608,7 @@
         configASSERT( pxNetworkBuffer != NULL );
         configASSERT( pxNetworkBuffer->pucEthernetBuffer != NULL );
 
-        BaseType_t xResult = pdPASS;
```

Comment:
I would rather see `pdTRUE` and `pdFALSE` in stead. The values are the same bu the meaning is: "the network buffer has been released".

Reply:
Thanks for the review @htibosch . Updated .